Gulpfile.coffee

```diff
@@ -82,7 +82,7 @@ gulp.task 'test', ['buildJS'], (done) ->
 
   return # Since this is async
 
-gulp.task 'tdd', ['build'],  (done) ->
+gulp.task 'tdd', (done) ->
```

Comment:
I'm not entirely sure we need a tdd task.  It is basically equivalent to the watchJS task.  I think one of them should get taken out. TDD does sound better IMO.